.github/CODEOWNERS

```
@@ -10,16 +10,18 @@
 /pages.tr/ @yutyo
 /pages.zh/ @blueskyson @einverne
 /pages.zh_TW/ @blueskyson
+/pages.ko/ @IMHOJEONG
 
-/contributing-guides/*.de.md @pixelcmtd @marchersimon
-/contributing-guides/*.es.md @navarroaxel @kant
-/contributing-guides/*.id.md @reinhart1010
-/contributing-guides/*.it.md @mebeim @yutyo @Magrid0
-/contributing-guides/*.nl.md @sebastiaanspeck
-/contributing-guides/*.pl.md @acuteenvy
-/contributing-guides/*.pt_BR.md @waldyrious @isaacvicente
-/contributing-guides/*.pt_PT.md @waldyrious
-/contributing-guides/*.ta.md @kbdharun
-/contributing-guides/*.tr.md @yutyo
-/contributing-guides/*.zh.md @blueskyson @einverne
-/contributing-guides/*.zh_TW.md @blueskyson
+/contributing-guides/_.de.md @pixelcmtd @marchersimon
+/contributing-guides/_.es.md @navarroaxel @kant
+/contributing-guides/_.id.md @reinhart1010
+/contributing-guides/_.it.md @mebeim @yutyo @Magrid0
+/contributing-guides/_.ko.md @IMHOJEONG
+/contributing-guides/_.nl.md @sebastiaanspeck
+/contributing-guides/_.pl.md @acuteenvy
+/contributing-guides/_.pt_BR.md @waldyrious @isaacvicente
+/contributing-guides/_.pt_PT.md @waldyrious
+/contributing-guides/_.ta.md @kbdharun
+/contributing-guides/_.tr.md @yutyo
+/contributing-guides/_.zh.md @blueskyson @einverne
+/contributing-guides/_.zh_TW.md @blueskyson
```

```suggestion
/contributing-guides/*.de.md @pixelcmtd @marchersimon
/contributing-guides/*.es.md @navarroaxel @kant
/contributing-guides/*.id.md @reinhart1010
/contributing-guides/*.it.md @mebeim @yutyo @Magrid0
/contributing-guides/*.ko.md @IMHOJEONG
/contributing-guides/*.nl.md @sebastiaanspeck
/contributing-guides/*.pl.md @acuteenvy
/contributing-guides/*.pt_BR.md @waldyrious @isaacvicente
/contributing-guides/*.pt_PT.md @waldyrious
/contributing-guides/*.ta.md @kbdharun
/contributing-guides/*.tr.md @yutyo
/contributing-guides/*.zh.md @blueskyson @einverne
/contributing-guides/*.zh_TW.md @blueskyson
```

We only have a few localized style guide pages and there won't be multiple files for a locale so the default wildcard pattern matching is fine.